### PR TITLE
Remove calendar view

### LINKED ISSUE #618: Remove 'Calendar' Page. When logging in, go to 'Schedule' page instead. Rename 'Schedule' to 'Dashboard'.
**Is your feature request related to a problem? Please describe.**
The Calendar page does very little for the user: they already have a calendar app that they're hooking up to us with, so it's redudant, and it's already a headache to maintain. We can scrap it and just drop the logged-in user into their Schedule page, which has a calendar view and immediately usable features for scheduling.

I just got sign-off from Design, so let's do it!

**Describe the solution you'd like**
Upon log-in, drop the user into the Schedule page instead of the Calendar page.
Remove the Calendar page.
Rename the Schedule page to Dashboard

**Describe alternatives you've considered**
We've discussed on-and-off the value of the Calendar page, and we all agreed to let it go for now and revisit if we ever need it.

### PR DESCRIPTION
**Pull request by devmount**
## Description of the Change

This change removes the following components:

- /elements/AppointmentListItem.vue
- /views/CalendarView.vue

Also the Schedule page got renamed to Dashboard as well as its url segment. To maintain compatibility to old links or sessions, the following redirects have been added:

- `/calendar/*` -> `/dashboard`
- `/schedule` -> `/dashboard`

## Applicable Issues

Closes #618

### CODE REVIEW

File: frontend/src/router.ts [hunk 1]
```diff
@@ -88,16 +86,17 @@ const routes: RouteRecordRaw[] = [
...
   },
   {
-    path: '/calendar',
-    redirect: { name: 'calendar' },
+    path: '/dashboard',
+    name: 'dashboard',
+    component: ScheduleView,
   },
   {
+    // This is just to auto-redirect old links. Can be removed in the future.
```
Comment (malini) [inline frontend/src/router.ts:97]: thanks for the context!

File: frontend/src/views/FirstTimeUserExperienceView.vue [hunk 2]
```diff
@@ -31,12 +31,12 @@ const refresh = inject('refresh');
...
   await refresh();
 
-  // If they're setup, boot them calendars
+  // If they're setup, boot them dashboard
   if (user.data.isSetup) {
     if (getPreferredTheme() === ColorSchemes.Dark) {
       document.documentElement.classList.add('dark');
     }
-    router.replace('calendar');
+    await router.replace('dashboard');
```
Comment (malini) [inline frontend/src/views/FirstTimeUserExperienceView.vue:39]: nice 🙌  I didn't realize this was a promise

File: frontend/src/views/HomeView.vue [hunk 3]
```diff
@@ -4,21 +4,21 @@ import InfoBox from '@/elements/home/InfoBox.vue';
...
+import { storeToRefs } from "pinia";
 
 const router = useRouter();
 
 const userStore = useUserStore();
 const { authenticated: isAuthenticated } = storeToRefs(userStore);
 
 const enter = () => {
-  router.push('/calendar');
+  router.push({ name: 'dashboard' });
```
Comment (malini) [inline frontend/src/views/HomeView.vue:15]: nice improvement!

**Review by malini — APPROVED** [review]
lgtm!

File: frontend/src/views/HomeView.vue (hunk 3 above)
Comment (devmount) [reply frontend/src/views/HomeView.vue:15]: Thanks! Yes it's much easier to change/refactor/redirect routes if we use route names only. I'll check if I find some more direct urls and fix them too.

**Comment by devmount** [comment]
@malini Wow thanks for the quick review!

**Review by aaspinwall — APPROVED** [review]
🧙🏼 yis

**Comment by devmount** [comment]
Thank y'all. Merging this now.

**Pull request merged**